### PR TITLE
Prepare release 1.0.3

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

File: azure-mgmt/README.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ Features
 This version of the Azure Management package bundle consists of the
 following packages. Follow the links for more information on each package.
 
--  `azure-mgmt-compute v0.20.0 <https://pypi.python.org/pypi/azure-mgmt-compute/0.20.0>`__
+-  `azure-mgmt-compute v0.20.1 <https://pypi.python.org/pypi/azure-mgmt-compute/0.20.0>`__
```
Comment (huguesv) [inline azure-mgmt/README.rst]: text is correct, but link is wrong

Comment (huguesv) [inline azure-mgmt/README.rst]: change azure-mgmt/setup.py (increment version)

**Comment by huguesv** [comment]
Update azure/setup.py
Update copyright year (2015) everywhere (including doc/conf.py)

**Comment by huguesv** [comment]
In azure/setup.py:
`'azure-servicemanagement-legacy==0.20.1'` should be `'azure-servicemanagement-legacy==0.20.2'`

**Comment by huguesv** [comment]
azure/readme.rst needs to be updated for version number changes of all 3 dependent libs.

**Comment by lmazuel** [comment]
Ready for 1.0.3!

**Pull request merged**